notebooks/add_a_single_event_to_a_sketch.ipynb

```diff
@@ -0,0 +1,187 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Timesketch_add_a_single_event_to_a_sketch.ipynb",
+      "provenance": [],
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/google/timesketch/blob/master/notebooks/Timesketch_add_a_single_event_to_a_sketch.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "E2meYbWjdnyR"
+      },
+      "source": [
+        "# Setup\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ZoKUE_mbwOyy"
+      },
+      "source": [
+        "# @title Install dependencies\n",
+        "# @markdown Only execute if not already installed and running a cloud runtime\n",
+        "!pip install -q timesketch_api_client\n",
+        "!pip install -q vt-py\n",
+        "!pip install -q nest_asyncio"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "cFNfts9KNOKi"
+      },
+      "source": [
+        "# @title Import libraries\n",
+        "# @markdown This cell will import all the libraries needed for the running of this colab.\n",
+        "\n",
+        "import altair as alt # For graphing.\n",
+        "import numpy as np   # Never know when this will come in handy.\n",
+        "import pandas as pd  # We will be using pandas quite heavily.\n",
+        "\n",
+        "from timesketch_api_client import config"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "_En91RhFQ5l8"
+      },
+      "source": [
+        "# @title Sketch Configuration\n",
+        "# @markdown Set the sketch ID for the rest of the notebook\n",
+        "\n",
+        "Sketch_id = 7193 #@param {type:\"integer\"}"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "c_rcGvf2yLvJ"
+      },
+      "source": [
+        "# @title Timesketch connnection\n",
+        "ts_client = config.get_client()\n"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "uO7KPHOrNh4_"
+      },
+      "source": [
+        "sketch = ts.get_sketch(Sketch_id)"
```

Comment:
why not combine these two in a single cell, that is get_client and get_sketch

Reply:
I'm also missing all the instructions in `get_client`, if people are running this for the first time, or use the cloud colab kernel they will not have a config already set, so they 'll need to enter some data to get information, probably better to include some instructions

Reply:
combined it into one cell. 

For the "more description" I am not 100 % sure I would agree here, so if we think the data requested when prompted is not enough, I opt we provide more info when get prompted instead of adding more explanation in every call we make to that function.

What do you think?

Reply:
well, pros and cons... ATM people are confused what to answer when prompted with `auth_mode`... 

I agree that there should be more explanations here, might need to add that, at least for these most common options.
but in this case here, since this is not using the demo server I'm fine with this as is for now...